src/main/resources/assets/js/components/AppListComponent.jsx

```diff
@@ -99,6 +100,11 @@ var AppListComponent = React.createClass({
               </span>
             </th>
             <th className="text-right">
+              <span className={headerClassSet}>
```

Comment:
Then we should remove the `headerClassSet`, so it is not displaying a false interaction, i.e. it is not clickable anymore